lib/core/presentation/pages/ai/ai_page.dart

```diff
@@ -110,36 +140,42 @@ class AIPageState extends State<AIPage> {
     return Scaffold(
       appBar: const LemonAppBar(),
       backgroundColor: colorScheme.primary,
-      body: Container(
-        decoration: BoxDecoration(
-          color: LemonColor.atomicBlack,
-          borderRadius: BorderRadius.circular(20.r),
-        ),
-        child: Column(
-          children: <Widget>[
-            Expanded(
-              child: ListView.builder(
-                controller: _scrollController,
-                padding: EdgeInsets.symmetric(
-                  vertical: 10.h,
-                  horizontal: 10.w,
+      body: SafeArea(
+        child: Container(
+          decoration: BoxDecoration(
+            color: LemonColor.atomicBlack,
+            borderRadius: BorderRadius.circular(20.r),
+          ),
+          child: Column(
+            children: <Widget>[
+              _buildChatList(),
+              Container(
+                decoration: BoxDecoration(color: Theme.of(context).cardColor),
+                child: AIChatComposer(
+                  textController: _textController,
+                  onSend: send,
+                  loading: _loading,
                 ),
-                itemCount: messages.length,
-                itemBuilder: (context, index) {
-                  return AIChatCard(message: messages[index]);
-                },
               ),
-            ),
-            Container(
-              decoration: BoxDecoration(color: Theme.of(context).cardColor),
-              child: AIChatComposer(
-                textController: _textController,
-                onSend: send,
-                loading: _loading,
-              ),
-            ),
-          ],
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildChatList() {
```

Comment:
remember Ron's comment on this ? we aligned that minimum  render method,  use Widget or put it directly in widget tree instead